src/plugins/intel_npu/tools/protopipe/README.md

```diff
@@ -60,6 +60,7 @@ log_level: INFO
 - `ol` - **Optional**. Output layer layout.
 - `iml` - **Optional**. Input model layout.
 - `oml` - **Optional**. Output model layout.
+- `shape` - **Optional**. Input layer shape.
```

Comment:
I believe it should be `reshape` to avoid confusion of what it actually does

Reply:
Ok. Will do.

Reply:
Please note that "-shape" is the command line argument used in benchmark app. Calling it "reshape" in protopipe might create confusion for people how search for an alternative to the benchmark app -shape option. 

I would stick to the same name benchmark app has chosen even if "reshape" was clearly a better name.

Reply:
Done